afids_utils/io.py

```diff
@@ -30,15 +67,15 @@
 
 
 def get_afid(
-    fcsv_path: PathLike[str] | str, fid_num: int
+    afids_fpath: PathLike[str] | str, fid_num: int
 ) -> NDArray[np.single]:
     """
     Extract specific fiducial's spatial coordinates
 
     Parameters
     ----------
-    fcsv_path : os.PathLike[str] | str
-        Path to .fcsv file to extract AFID coordinates from
+    afids_path : os.PathLike[str] | str
```

Comment:
This doesn't match up with the actual parameter name, and I don't think we need to repeat the type annotation here.

Reply:
Oops, good catch. The repeated type annotation here was mainly for the docs, but can remove.

Reply:
I'm pretty sure there's a sphinx extension (or built-in) that picks up the type hints in the signature for the docs, FWIW

Reply:
Ahh you are right - `autodoc_docstring_signature`. Would you be opposed to leaving that to a separate PR (can make an issue for it so it's not forgotten).

Reply:
That's fine by me!